packages/react-native/src/primitives/Tabs/styles.ts

```diff
@@ -1,14 +1,30 @@
-import { StyleSheet, TextStyle } from 'react-native';
+import { StyleSheet, ViewStyle } from 'react-native';
 
 import { StrictTheme } from '../../theme';
-import { TabsStyles } from './types';
+import { TabsProps, TabsStyles } from './types';
 
-export const getThemedStyles = (theme: StrictTheme): TabsStyles => {
+export const getThemedStyles = (
+  theme: StrictTheme,
+  indicatorPosition?: TabsProps['indicatorPosition']
+): TabsStyles => {
   const {
     components,
-    tokens: { colors, fontSizes, fontWeights, opacities, space },
+    tokens: { colors, fontSizes, fontWeights, opacities, space, borderWidths },
   } = theme;
 
+  const selectedTabBorderStyles: ViewStyle = {};
+  const tabBorderStyles: ViewStyle = {};
+
+  if (indicatorPosition && indicatorPosition === 'top') {
+    selectedTabBorderStyles.borderTopColor = colors.brand.primary[80];
+    tabBorderStyles.borderTopWidth = borderWidths.medium;
+    tabBorderStyles.borderTopColor = colors.border.secondary;
+  } else {
+    selectedTabBorderStyles.borderBottomColor = colors.brand.primary[80];
+    tabBorderStyles.borderBottomWidth = borderWidths.medium;
+    tabBorderStyles.borderBottomColor = colors.border.secondary;
+  }
```

Comment:
Can be addressed outside of this PR but a little iffy on adopting this pattern in to component styling